use std::result;

fn main() {
    let result_error: result::Result<(), ErrorType>;
}